lib/PuppeteerSharp/Page.cs

```diff
@@ -874,5 +875,8 @@ private void OnDialog(PageJavascriptDialogOpeningResponse message)
         #region IDisposable
         public void Dispose() => CloseAsync();
         #endregion
+        #region IEquatable
+        public bool Equals(Page other) => MainFrame.Id == other.MainFrame.Id;
```

Comment:
Are we sure about this equality? Where are we using this?

Reply:
i guess it wasn't needed, forgot why I added it 😆